.gitignore

```diff
@@ -17,3 +17,6 @@
 *.log
 *.enc
 *.dec
+
+scripts/.env
+scripts/volumes
```

Comment:
I prefer if we prefix all the ignored paths with `/`

Reply:
Ok. Any real life experience reason ?

Reply:
I had once a file in another sub-directory that was ignored by mistake because it matches a rule with no `/` prefix, and it makes the CI red, but it was hard to debug.

More generally, we may want to ignore some files, and but have a copy of them in a specific directory for testing.

Reply:
Thanks for the intel